include/fs/fs.h

```diff
@@ -66,7 +66,14 @@ enum {
  * This flag has no impact in user-defined mount structures.
  */
 #define FS_MOUNT_FLAG_AUTOMOUNT BIT(2)
-
+/** Flag requests file system to use Disk Access API. When a file system supports
```

Comment:
Flag to force the file system to use the Disk Access API. When this flag is
set prior to fs_mount the file system should use the Disk Access API. When the flag is not set prior to fs_mount the file system can use its preferred API.
When a file system only supports the Disk Access API it should set the flag.
`

Reply:
Little bit reworded your suggestion, but generally followed the comment.

Reply:
> 

NP, looks good to me.